parm/config/config.gldas

```diff
@@ -12,9 +12,7 @@ export GLDASSH=$HOMEgfs/scripts/exgdas_gldas.sh.ecf
 export gldas_spinup_hours=72
 
 
-if [ $machine = "WCOSS_C" -o $machine = "WCOSS_DELL_P3" ]; then
-    export CPCGAUGE=/gpfs/dell2/emc/modeling/noscrub/Fanglin.Yang/stat/cpcUniGauge
-elif [ $machine = "HERA" ]; then
+if [ $machine = "HERA" ]; then
     export CPCGAUGE=/scratch1/NCEPDEV/global/Fanglin.Yang/stat/cpcUniGauge
 fi
 
```

Comment:
The CPC gauge files are in the Hera GDA just like in the WCOSS GDA so this isn't necessary anymore.

Reply:
already removed in the new branch